pkg/handlers/landing.go

```diff
@@ -0,0 +1,55 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+type LandingHandler struct {
+}
+
+type status string
+
+const (
+	draft status = "DRAFT"
+)
+
+// Profile is a simple struct for testing the API
+type Profile struct {
+	Name         string
+	SystemOwners []string
+	CreatedAt    time.Time
+	UpdatedAt    time.Time
+	Status       status
+}
+
+// HandleLanding is the handler for the root path
+func (h LandingHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	sampleProfile := Profile{
+		Name:         "My Favorite Project",
+		SystemOwners: []string{"admin@example.com", "operations@example.com"},
+		CreatedAt:    time.Now(),
+		UpdatedAt:    time.Now(),
+		Status:       draft,
+	}
+	js, err := json.Marshal(sampleProfile)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	enableCors(&w)
+
+	_, err = w.Write(js)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+}
+
+func enableCors(w *http.ResponseWriter) {
```

Comment:
Seems like maybe this is a middleware responsibility rather than in the handler.

Also, can we look into how we should be dealing with CORS in general? I see that the okta docs say to enable it here and in the console, but I'm wondering if there's some other protection we have to be applying. Vector I'm thinking of is someone just dropping the widget directly into another application, then forcing a user to send a request to our APIs with the JWT token. This seems like we should ask Okta folks the best route with this.